sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobServiceSASSignatureValues.java

```diff
@@ -33,7 +35,7 @@
  * for additional samples.</p>
  */
 
-final class BlobServiceSASSignatureValues {
+public final class BlobServiceSASSignatureValues {
```

Comment:
Another public API

Reply:
Not entirely certain if this should be moved into an `implementation` package or maintain becoming public API given all the documentation and methods in the class make it seem that it should be usable by a consumer of the library.

Reply:
There is immediate work that needs to be done in regards to SAS token generation, will handle in that PR.

https://github.com/Azure/azure-sdk-for-java/issues/5541